src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -810,6 +808,7 @@ export class EditDataGridPanel extends GridParentComponent {
 				for (let i = 0; i < dataSet.columnDefinitions.length; i++) {
 					this.columnNameToIndex[this.dataSet.columnDefinitions[i].name] = i;
 				}
+				this.onResize();
```

Comment:
the grid seems to be created the correct size as far as I can tell.  why is this call needed here?